### PR TITLE
Vep pli update

### PR DESCRIPTION
**Pull request by Lucpen**
## PR checklist

- [x] Updated pLi config
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/raredisease/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/raredisease _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test_one_sample,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: conf/modules/annotate_genome_snvs.config [hunk 1]
```diff
@@ -77,10 +77,10 @@ process {
 
         withName: '.*ANNOTATE_GENOME_SNVS:ENSEMBLVEP_SNV' {
             ext.prefix = { "${vcf.simpleName}_rohann_vcfanno_filter_vep" }
-            ext.args   = [
+            ext.args = { [
                 '--dir_plugins vep_cache/Plugins',
                 '--plugin LoFtool,LoFtool_scores.txt',
-                '--plugin pLI,pLI_values_107.txt',
+                "--plugin pLI,pLI_values_${params.vep_cache_version}.txt",
```
Comment (Lucpen) [inline conf/modules/annotate_genome_snvs.config:83]:
```suggestion
                "--plugin pLI,pLI_values.txt",
```

File: conf/modules/annotate_mobile_elements.config [hunk 2]
```diff
@@ -40,7 +40,7 @@ process {
         ext.args   = { [
             '--dir_cache vep_cache',
             '--dir_plugins vep_cache/Plugins',
-            '--plugin pLI,pLI_values_107.txt',
+            "--plugin pLI,pLI_values_${params.vep_cache_version}.txt",
```
Comment (Lucpen) [inline conf/modules/annotate_mobile_elements.config:43]:
```suggestion
            "--plugin pLI,pLI_values.txt",
```

File: conf/modules/annotate_mt_snvs.config [hunk 3]
```diff
@@ -18,10 +18,10 @@
 process {
     if (!params.skip_mt_annotation) {
         withName: '.*ANNOTATE_MT_SNVS:ENSEMBLVEP_MT' {
-            ext.args   = [
+            ext.args = { [
                 '--dir_plugins vep_cache/Plugins',
                 '--plugin LoFtool,LoFtool_scores.txt',
-                '--plugin pLI,pLI_values_107.txt',
+                "--plugin pLI,pLI_values_${params.vep_cache_version}.txt",
```
Comment (Lucpen) [inline conf/modules/annotate_mt_snvs.config:24]:
```suggestion
                "--plugin pLI,pLI_values.txt",
```

File: conf/modules/annotate_structural_variants.config [hunk 4]
```diff
@@ -43,18 +43,18 @@ process {
         }
 
         withName: '.*ANNOTATE_STRUCTURAL_VARIANTS:ENSEMBLVEP_SV' {
-            ext.args   = [
+            ext.args = { [
                 '--dir_cache vep_cache',
                 '--dir_plugins vep_cache/Plugins',
-                '--plugin pLI,pLI_values_107.txt',
+                "--plugin pLI,pLI_values_${params.vep_cache_version}.txt",
```
Comment (Lucpen) [inline conf/modules/annotate_structural_variants.config:49]:
```suggestion
                "--plugin pLI,pLI_values.txt",
```

File: conf/modules/annotate_genome_snvs.config [hunk 5]
```diff
@@ -77,10 +77,10 @@ process {
 
         withName: '.*ANNOTATE_GENOME_SNVS:ENSEMBLVEP_SNV' {
             ext.prefix = { "${vcf.simpleName}_rohann_vcfanno_filter_vep" }
-            ext.args   = [
+            ext.args = { [
                 '--dir_plugins vep_cache/Plugins',
                 '--plugin LoFtool,LoFtool_scores.txt',
-                '--plugin pLI,pLI_values_107.txt',
+                "--plugin pLI,pLI_values.txt",
```
Comment (Lucpen) [inline conf/modules/annotate_genome_snvs.config:83]:
```suggestion
                '--plugin pLI,pLI_values.txt',
```

File: conf/modules/annotate_mobile_elements.config [hunk 6]
```diff
@@ -40,7 +40,7 @@ process {
         ext.args   = { [
             '--dir_cache vep_cache',
             '--dir_plugins vep_cache/Plugins',
-            '--plugin pLI,pLI_values_107.txt',
+            "--plugin pLI,pLI_values.txt",
```
Comment (Lucpen) [inline conf/modules/annotate_mobile_elements.config:43]:
```suggestion
            '--plugin pLI,pLI_values.txt',
```

File: conf/modules/annotate_mt_snvs.config [hunk 7]
```diff
@@ -18,10 +18,10 @@
 process {
     if (!params.skip_mt_annotation) {
         withName: '.*ANNOTATE_MT_SNVS:ENSEMBLVEP_MT' {
-            ext.args   = [
+            ext.args = { [
                 '--dir_plugins vep_cache/Plugins',
                 '--plugin LoFtool,LoFtool_scores.txt',
-                '--plugin pLI,pLI_values_107.txt',
+                "--plugin pLI,pLI_values.txt",
```
Comment (Lucpen) [inline conf/modules/annotate_mt_snvs.config:24]:
```suggestion
                '--plugin pLI,pLI_values.txt',
```

File: conf/modules/annotate_structural_variants.config [hunk 8]
```diff
@@ -43,18 +43,18 @@ process {
         }
 
         withName: '.*ANNOTATE_STRUCTURAL_VARIANTS:ENSEMBLVEP_SV' {
-            ext.args   = [
+            ext.args = { [
                 '--dir_cache vep_cache',
                 '--dir_plugins vep_cache/Plugins',
-                '--plugin pLI,pLI_values_107.txt',
+                "--plugin pLI,pLI_values.txt",
```
Comment (Lucpen) [inline conf/modules/annotate_structural_variants.config:49]:
```suggestion
                '--plugin pLI,pLI_values.txt',
```

**Review by jemten — APPROVED** [review]
👍

**Comment by jemten** [comment]
merging since the only failures is on nf-core linting. The warnings are unrelated to this PR and will hopefully be fixed with a new version of the nf-core TEMPLATE

**Pull request merged**